package.json

```diff
@@ -26,6 +33,7 @@
     "gulp-streamify": "^1.0.2",
     "gulp-stylus": "^2.1.0",
     "gulp-uglify": "^1.5.1",
+    "react": "^0.14.7",
```

Comment:
Are we using the React brought in by the `devDependencies` at all? We're still relying on the bower version, and the new exposify addition doesn't seem to depend on this being listed in devDependencies at all.

I'm also confused by the addition of React in `peerDependencies`. This just means that consumers who pull in Geniblocks via npm will also pull in React, but at the moment, at least, we're agnostic about how the consumer pulls in React -- they could be using bower, like we do for our examples, or something else.

(Ultimately my guess is that this will all start becoming clearer once we create the Geniverse 2.0 application, which is something I feel like we should be getting to soon.)